src/login/lclif.c

```diff
@@ -293,6 +293,7 @@ static bool lclif_send_server_list(struct login_session_data *sd)
 	packet->packet_id = HEADER_AC_ACCEPT_LOGIN;
 #else
 	packet->packet_id = HEADER_AC_ACCEPT_LOGIN2;
+	login->generate_token(sd, packet->auth_token);
```

Comment:
Is this needed? You can use postHook and create a method in the plugin itself.

Reply:
AFAIK, unless we override the entire function with a custom one, we cannot hook into the packet info to send the token, as the token is not currently being sent.
Otherwise the plugin would just send this packet again after herc has sent it as response to the user logging in, and I don't know if the client would update the token in that case.

Reply:
ah yes, sorry